src/helics/apps/MultiBroker.cpp

```diff
@@ -84,6 +85,11 @@ static std::unique_ptr<CommsInterface> generateComms(const std::string &type, co
 		case core_type::ZMQ:
 #if HELICS_HAVE_ZEROMQ!=0
 			comm = std::make_unique<zeromq::ZmqComms>();
+#endif
+			break;
+		case core_type::ZMQ_TEST:
```

Comment:
Do we want to leave this as ZMQ_TEST or is another short name more descriptive

Reply:
I would prefer to change it to "ZMQ_SS" similar to "TCP_SS"